components/cd-bullet-list/README.md

```diff
@@ -0,0 +1,18 @@
+# Bullet list
+
+## Purpose and Usage
+A list of text or links with a chevron icon as bullet.
+
+## Caveats
+None
+
+### Variants
+The default bullet list uses the SVG element data URI as a background image. This means the fill colour can still be 
+edited in CSS.
+
+The other variant `--icon` is when the SVG element markup is added to the list.
+
+```
+cd-bullet-list--icon
```

Comment:
the other READMEs I saw prefixed variants with dots to indicate that it's a CSS class